schema/2.0.0/seurat_encoding.md

```diff
@@ -0,0 +1,226 @@
+
+# Schema - Seurat encoding
+
+Contact: acarr@chanzuckerberg.com
+
+Document Status: _In progress_
+
+Schema version: 2.0.0
+
+
+## Overview
+
+All data submitted to the [cellxgene data portal](https://cellxgene.cziscience.com/) is automatically converted to a Seurat V3 object that can be loaded by the R package [Seurat](https://satijalab.org/seurat/). The conversion is done through [sceasy](https://github.com/cellgeni/sceasy).
+
+This document describes such Seurat encoding for the converted data and it mirrors the data [schema](./schema.md).
+
+## Encoding
+
+The format used to store the Seurat object is [`.rds`](https://stat.ethz.ch/R-manual/R-devel/library/base/html/readRDS.html) which provides an interface for storing individual R objects on disk.
+
+A `local.rds` file downloaded from the cellxgene data portal can be read into an R session by doing the following:
```

Comment:
To be consistent with other documentation, it should be cellxgene Data Portal. 

~by doing the following:~

with the following code.

Would it be helpful to have a notebook to illustrate the encoding?

Reply:
It _could_ be helpful. But probably as a separate document.